tests/spec/fixtures/test-config-empty.xml

```diff
@@ -1,26 +1,25 @@
-<?xml version='1.0' encoding='utf-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I do not (yet) see the relevance of this update in terms of testing. Any other thoughts here?